src/utils/DateUtils.js

```diff
@@ -21,8 +21,9 @@ function stringDateFiveDaysBefore(date) {
   return toStringDate(d);
 }
 
-function isInPast(date) {
-  const d = moment(date);
+function isInPast(date, start) {
+  const str = date.concat(', ', start);
+  const d = moment(str, 'YYYY-MM-DD, HH:mm');
   return d.isBefore();
```

Comment:
Can you help me understand what this is doing? on line 26, what sort of moment is created? Does it represent a range of time?

Reply:
From what I understand, there are a few ways to construct a moment, including giving it a date, or giving it a string and another string describing the format you’re using.  The simplest way I could think of to construct a moment that had both date and time was to just concatenate the two strings together and construct the moment from that (the momentjs documentation is also pretty good!)

Reply:
Okay gotcha I wasn't seeing you pass in this.startTime so I was just imagining "start" as another date. This makes plenty of sense now. Thanks!